php-practice_1&2/php-practice_1.php

```diff
@@ -1,41 +1,132 @@
 <?php
 // Q1 変数と文字列
-
+$name = '「中田」';
+echo  '私の名前は' .$name . 'です。';
```

Comment:
[nits]
.などの演算時の両側には半角スペースを開けてください！（他にも何個かあるので、見直してみましょう！）
```suggestion
echo  '私の名前は' . $name . 'です。';